waitress/pantry/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Let's register the Pantry model here so we can be able to access it inside `Django Admin`.